api/jobs_test.go

```diff
@@ -46,43 +44,6 @@ func TestJobs_Register(t *testing.T) {
 	}
 }
 
-func TestJobs_Parse(t *testing.T) {
```

Comment:
to call it out for future reviews, this and many removed tests were moved to `apitests/`